geojson/src/commonMain/kotlin/org/maplibre/spatialk/geojson/Position.kt

```diff
@@ -115,6 +116,5 @@ public class Position(public val coordinates: DoubleArray) {
     public fun json(): String = coordinates.jsonJoin()
 }
 
-@Suppress("MagicNumber")
 public val Position.hasAltitude: Boolean
-    get() = coordinates.size == 3
+    get() = coordinates.size >= 3
```

Comment:
drive-by bug fix: if additional elements are provided, the third is still the altitude